runtime/isp_run_app.py

```diff
@@ -66,10 +66,10 @@ def compileMissingPex(policy_dir, pex_path, sim, arch, extra):
     return True
 
 
-def compileMissingPolicy(policies, global_policies, debug):
+def compileMissingPolicy(policies, global_policies, output_dir, debug):
     logger.info("Attempting to compile missing policy")
     args = ["isp_install_policy",
-            "-O", os.path.join(isp_prefix, "policies"),
+            "-O", output_dir,
```

Comment:
Shouldn't the output_dir contains the pouches subdirectory? As in os.path.join(output_dir, "policies")?

Reply:
That directory does get created by `isp_install_policy`.